### DISCUSSION TITLE
Memory for individual chats

### DISCUSSION THREAD

**Discussion by lijackcoder** [open-webui/open-webui#8029] [Ideas]
# Feature Request

instead of the memory feature applying to all chats, have an extra option for only a specific chat

**Comment by codewithtyler** [comment]
This would be great. I have in my memory some info about my current GitHub sponsor because I will sometimes ask questions about their software. But most often I will ask random software dev questions and it's annoying that it starts asking me mid conversation about what my question has to do with said sponsor. It has literally nothing to do with the sponsor. I gave it that information so that it has additional context when needed. That doesn't mean I want every conversation to be about said GitHub sponsorship.
